### PR TITLE
Dependency upgrade

### PR DESCRIPTION
**Pull request by dalSara**
Updated FE dependencies. 

Please check throughly as some of those are major updates.

### CODE REVIEW

File: source/static-site/pages/pages.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // NOTE: Do not edit this file. It is built by running `./scripts/create-static-site-files.js`
```
Comment (arcuri82) [inline source/static-site/pages/pages.js:1]: should generated files really be added to Git?

File: source/static-site/pages/paths.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 // NOTE: Do not edit this file. It is built by running `./scripts/create-static-site-files.js`
```
Comment (arcuri82) [inline source/static-site/pages/paths.js:1]: see comment above

File: source/static-site/pages/pages.js (hunk 1 above)
Comment (dalSara) [reply source/static-site/pages/pages.js:1]: added to gitignore

File: source/static-site/pages/paths.js (hunk 2 above)
Comment (dalSara) [reply source/static-site/pages/paths.js:1]: added to gitignore

**Pull request closed without merging**